wondrous-app/components/profile/modals/index.tsx

```diff
@@ -1,17 +1,33 @@
-import React, { useCallback, useEffect, useRef, useState, useContext } from 'react';
+import React, { useEffect, useState } from 'react';
 import Modal from '@mui/material/Modal';
 import { CircularProgress } from '@mui/material';
-
-import { format, formatDistance } from 'date-fns';
-import { useInView } from 'react-intersection-observer';
 import { GET_ORG_PODS, GET_ORG_USERS } from 'graphql/queries/org';
 import { GET_POD_USERS } from 'graphql/queries/pod';
 import { useLazyQuery } from '@apollo/client';
-import { TaskModalBaseCard, TaskSubmissionHeaderCreatorText } from '../../Common/Task/styles';
-import { TabContainer, Tab, TabContainerText, PodExplainerText } from './styles';
-import { DefaultProfilePicture, PodWrapper, Title, UserProfilePicture, UserWrapper } from './styles';
+import Image from 'next/image';
+
+import { TaskModalBaseCard } from '../../Common/Task/styles';
```

Comment:
can we use absolute imports here? e.g import {TaskModalBaseCard} from 'components/Common/Task/styles'

Reply:
done